packages/marshal/src/marshal.js

```diff
@@ -300,6 +300,15 @@ const makeRemotableProto = (oldProto, allegedName) => {
   });
 };
 
+/**
+ * Throw if val is not the correct shape for the prototype of a Remotable.
+ *
+ * TODO: It would be nice to typedef this shape and then declare that this
+ * function asserts it, but we can't declare a type with PASS_STYLE from JSDoc.
+ *
+ * @param {{ [PASS_STYLE]: string, [Symbol.toStringTag]: string, toString: () =>
```

Comment:
odd use of whitespace. I'd rather see each member on its own line, for consistency with prettify

Reply:
I'm a victim of my own auto-wrap-comments tooling.